package.json

```diff
@@ -50,21 +50,24 @@
   },
   "typings": "index.d.ts",
   "devDependencies": {
+    "bluebird": "^3.5.2",
```

Comment:
I only found the use of it in tests, and I think native Promise should work there, is it really needed?